src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=postgres
+spring.datasource.password=redert77
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.security.user.name=admin
+spring.security.user.password=geeksforgeeks
```

Comment:
What percentage of the project did you copy from Geeks4Geeks?

Reply:
The authentication. I had initially made my own system but I soon realized it was very flawed so I went online to find common Java practices when it comes to building auth/jwt, this is mostly due to me being unfamiliar with the syntax and java feeling very foreign in general, but even now I feel a lot more comfortable and will need to depend on other code less.